docs/upgrading.md

```diff
@@ -12,6 +12,10 @@ Migration guides contain information regarding the new features introduced in ea
 
 ## General Information
 
+Upgrading is not advised on running apps. Stop them before with `dokku ps:stop <app>`.
```

Comment:
I would think so. I wouldn't upgrade on a running "production" host. Specifically when we require a particular version of docker that could potentially cause that package to be upgraded would cause the docker daemon to restart. Thus, resulting in downtime.